src/App.css

```diff
@@ -1 +1,3 @@
-
+* {
+  margin-left: 0;
+}
```

Comment:
- [ ] As requested by the previous reviewer, kindly remove `App.js` as it is boilerplate code.